public/javascripts/controllers/admin-ctrl.js

```diff
@@ -1,8 +1,9 @@
 angular.module('jasmic.controllers')
-    .controller('AdministratorCtrl', ['$scope','$mdDialog', '$mdToast','UsersFactory','CropsFactory','SuppliersFactory',
+    .controller('AdministratorCtrl', ['$scope','$mdDialog', '$mdToast','$window','UsersFactory','CropsFactory','SuppliersFactory',
     'UserFactory','CropFactory','SupplierFactory', 'EmailFactory', 'UserProfileFactory', 'ParishesFactory',
```

Comment:
@matjames007 I encountered an issue after updating a buyer. The screen would turn grey but the dialog box was not seen. In testing the branch, you can remove the $window.scroll line to see if you get encounter this behaviour.